dashboard/app/models/census/census_summary.rb

```diff
@@ -155,10 +155,13 @@ def self.summarize_submission_data(school, school_year, high_school, k8_school,
   end
 
   def self.summarize_ap_data(school, school_year, audit)
-    ap_offerings = school.ap_school_code.try(:ap_cs_offering) || []
-    ap_offerings_this_year = ap_offerings.select {|o| o.school_year == school_year}
-    ap_offerings_this_year.each do |offering|
-      audit[:ap_cs_offerings].push(offering.id)
+    ap_offerings_this_year = []
```

Comment:
Once we resolved the ActiveRecord error in the `ap_school_code` factory, we resolved a real test failure that resulted from the prior assumption that each school had only one `ap_school_code`. Now a school can have a different `ap_school_code` per year.